resources/views/settings/tags.blade.php

```diff
@@ -94,3 +102,15 @@
 </div>
 
 @endsection
+<script>
+  function toggleEditInput(editIcon) {
+    var tableRow = editIcon.closest('.table-row');
```

Comment:
```suggestion
    let tableRow = editIcon.closest('.table-row');
```
The `let` is recommended when declaring variables
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let